core-kotlin-modules/core-kotlin-6/README.md

```diff
@@ -7,3 +7,4 @@
 - [How to Invoke External Command From Within Kotlin](https://www.baeldung.com/kotlin/invoke-external-command)
 - [Kotlin Equivalent to Java String Arrays](https://www.baeldung.com/kotlin/java-string-arrays-equivalent)
 - [Kotlin Bit Operations](https://www.baeldung.com/kotlin/bit-operations)
+- [Kotlin Error "Accidental override: The following declarations have the same JVM signature"]()
```

Comment:
Let's delete this line - it'll be added back in after this article is published, and will fix the conflicts.

```suggestion
```